### PR TITLE
Pull collection

### PR DESCRIPTION
**Pull request by damaggu**
couple of things, still in the works

### CODE REVIEW

File: samri/pipelines/nipype_based/diagnostics.py [hunk 1]
```diff
@@ -149,15 +150,25 @@ def diagnose(measurements_base,
 			(s_bru2nii, datasink, [('nii_file', 'anat')]),
 			])
 
+	#TODO: realigner  optional / als flag (wenn spm) ansonsten mandatory, slicetiming nicht spm
+	#TODO: realign vs. not realign, ICA geht verloren auf neurostars posten
 	if realign:
```
Comment (TheChymera) [inline samri/pipelines/nipype_based/diagnostics.py]: Nested if statements are unhandy and here also clutter the function parameter space (I am very guilty of that myself :-/ ). In this case the conditional is better addressed with realign="one string or another" and elif statements.

File: samri/pipelines/nipype_based/diagnostics.py [hunk 2]
```diff
@@ -149,15 +150,25 @@ def diagnose(measurements_base,
 			(s_bru2nii, datasink, [('nii_file', 'anat')]),
 			])
 
+	#TODO: realigner  optional / als flag (wenn spm) ansonsten mandatory, slicetiming nicht spm
+	#TODO: realign vs. not realign, ICA geht verloren auf neurostars posten
```
Comment (TheChymera) [inline samri/pipelines/nipype_based/diagnostics.py]: Please translate to english and add link to forum thread https://neurostars.org/t/nipy-spacetimerealigner-problem/337/7

**Review by TheChymera — CHANGES_REQUESTED** [review]

**Comment by TheChymera** [comment]
Awesome PR!

I am working quite intensively on SAMRI these days (as you can tell by the commit frequency). I am going to sleep now (been working all night) so there won't be new commits in the next few hours. Would be great if you could make those small corrections soon, so I can merge right after I wake up and we can avoid any conflicts.

File: samri/pipelines/nipype_based/diagnostics.py (hunk 2 above)
Comment (damaggu) [reply samri/pipelines/nipype_based/diagnostics.py]: done

File: samri/pipelines/nipype_based/diagnostics.py (hunk 1 above)
Comment (damaggu) [reply samri/pipelines/nipype_based/diagnostics.py]: and done

**Pull request merged**